dashboard/db/schema.rb

```diff
@@ -1372,14 +1372,17 @@
   end
 
   create_table "teacher_feedbacks", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
-    t.text     "comment",     limit: 65535
+    t.text     "comment",                  limit: 65535
```

Comment:
Should the extra space added here be worrisome?

Reply:
nope, when you add/remove columns on the schema, the spacing is updated so that everything aligns